charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
It would be nice to add some comments here that this file is auto generated using values from `helmCharts` fields of the add-ons kustomization.yaml.tmpl. This file can not be directly edited. respective add-ons kustomization template need to change and then `make generate-helm-configmap` or using pre-commit hook.

Reply:
pre-commit hook is already implemented.